examples/protocols/displayport/displayport-src.stanza

```diff
@@ -0,0 +1,143 @@
+#use-added-syntax(jitx)
+defpackage jsl/examples/protocols/displayport/displayport-src :
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/symbols/SymbolDefn
+  import jsl/symbols/box-symbol
+  import jsl/landpatterns/packages
+  import jsl/landpatterns/BGA
+
+  import jsl/design/settings
+  import jsl/si/helpers
+  import jsl/pin-assignment
+  import jsl/bundles
+  import jsl/protocols/displayport
+
+doc: \<DOC>
+@brief DisplayPort dummy component
+The component defined in this file is an example of how to enable a DisplayPort interface
+on a component. The component demonstrates the use of JSL land-patterns and symbols.
+<DOC>
+
+val body = PackageBody(
+  width = 15.0 +/- 0.1,
+  length = 15.0 +/- 0.1,
+  height = 0.71 +/- [0.06, 0.0]
+)
+
+val BGA-pkg = BGA(
+  num-leads = 49,
```

Comment:
Leads is wrong here - should be 64. Probably wrong in PCIe as well.

Reply:
Will fix.